src/open_inwoner/templates/pages/profile/contacts/list.html

```diff
@@ -3,127 +3,143 @@
+                            {% endrender_form %}
+                        </div>
+                    {% endwith %}
+                {% else %}
+                    <ul class="approval__list">
+                        {% for approval in pending_approvals %}
+                            <li class="approval__list-item">
+                                <p class="p">{{ approval.get_full_name }}</p>
+                                {% url 'accounts:contact_approval' uuid=approval.uuid as approval_url %}
+                                {% render_form form=None method="POST" form_action=approval_url id="approval_form" spaceless=True show_notifications=True %}
+                                    {% csrf_token %}
+                                    {% button_row %}
+                                        {% button text=_("Weigeren") name="contact_reject" value="reject" icon="close" icon_position="before" extra_classes="button--danger-borderless" %}
+                                        {% button text=_("Accepteren") name="contact_approve" value="approve" icon="check" icon_position="before" %}
+                                    {% endbutton_row %}
+                                {% endrender_form %}
+                            </li>
+                        {% endfor %}
+                    </ul>
+                {% endif %}
             </div>
-            {% endwith %}
-        {% else %}
-            <ul class="approval__list">
-                {% for approval in pending_approvals %}
-                    <li class="approval__list-item">
-                        <p class="p">{{approval.get_full_name}}</p>
-                        {% url 'accounts:contact_approval' uuid=approval.uuid as approval_url %}
-                        {% render_form form=None method="POST" form_action=approval_url id="approval_form" spaceless=True show_notifications=True %}
-                            {% csrf_token %}
-                            {% button_row %}
-                                {% button text=_("Weigeren") name="contact_reject" value="reject" icon="close" icon_position="before" extra_classes="button--danger-borderless" %}
-                                {% button text=_("Accepteren") name="contact_approve" value="approve" icon="check" icon_position="before" %}
-                            {% endbutton_row %}
-                        {% endrender_form %}
-                    </li>
-                {% endfor %}
-            </ul>
-        {% endif %}
-    </div>
-    {% endwith %}
-{% endif %}
+        {% endwith %}
+    {% endif %}
 
-{% form id="contact-filter" form_object=form method="GET" columns=2 no_actions=True horizontal=True spaceless=True autosubmit=True form_action=form_action %}
+    <div class="contacts">
+        {% render_form id="contact-filter" form=form method="GET" no_actions=True horizontal=True spaceless=True autosubmit=True form_action=form_action class="form form--columns-2 form--spaceless form--autosubmit form--horizontal" %}
```

Comment:
The `class` variable is not part of the template tag definition. The classes you define inside this variable are automatically generated by the template tag since you have passed `spaceless=True`, `autosubmit=True`, `horizontal=True`.

Reply:
Thanks! I have no idea why I did it like this...